crates/node/service/src/service/standard/mod.rs

```diff
@@ -101,27 +93,17 @@ impl ValidatorNodeService for RollupNode {
         if self.network_disabled {
             return Ok(None);
         }
-        let gossip_addr = self.gossip_address.ok_or_else(|| {
-            RollupNodeError::Network(NetworkBuilderError::GossipDriverBuilder(
-                GossipDriverBuilderError::GossipAddrNotSet,
-            ))
-        })?;
-        let disc_addr = self.discovery_address.ok_or_else(|| {
-            RollupNodeError::Network(NetworkBuilderError::DiscoveryDriverBuilder(
-                Discv5BuilderError::ListenConfigNotSet,
-            ))
-        })?;
-        let keypair = self.keypair.clone().unwrap_or_else(Keypair::generate_secp256k1);
-
+        if self.net_config.is_none() {
+            warn!(
+                target: "rollup_node",
+                "No network configuration provided, skipping network initialization"
```

Comment:
I think we should call this p2p. Network configuration is a bit opaque, means quite a few things. (Questions a user might ask themselves: Is network configuration for L2 network config? L1 network config? What network are we talking about?)